lib/packages/libbsdgames2.rb

```diff
@@ -0,0 +1,85 @@
+require 'package'
+
+class Bsdgame < Package
+  $_bsdgame_homepage = 'https://www.polyomino.org.uk/computer/software/bsd-games/'
```

Comment:
Use class variable? (only can be read in relative classes)
`@@_bsdgame_homepage = '...'`

Reply:
I tried that and it didn't work. How would you go about that?